### PR TITLE
Update: Use denali link and hover colors

### PR DESCRIPTION
**Pull request by jkusa**
## Description

Use denali link and hover colors

## Proposed Changes

Use denali link and hover colors based the repo's current css variables:

```css
  --links-text-color: var(--color-brand-600);
  --links-hover-text-color: var(--color-brand-700);
```
https://github.com/denali-design/denali-site/blob/96fee64f00208c3993fb9a5c240b42c4ae2c86d5/assets/denali.css#L177-L178

## Screenshots

Screen Shot 2020-03-16 at 4 04 40 PM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/core/app/styles/navi-core/variables/colors.less [hunk 1]
```diff
@@ -97,7 +97,9 @@
 @navi-disabled-element-background-color: @navi-gray-200;
 
 // TODO update names and move to theme.less
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/core/app/styles/navi-core/variables/colors.less]: Not sure the value of this comment. I'll remove it.

Comment (jkusa) [inline packages/core/app/styles/navi-core/variables/colors.less]:
```suggestion
```

File: packages/app/app/styles/components/nav-bar.less [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
     border-radius: 5px;
-    color: @link-blue;
+    color: @denali-link;
     display: flex;
     padding: 5px 10px;
     text-align: center;
 
     &:hover {
-      background: @link-blue;
+      background: @denali-link;
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/app/app/styles/components/nav-bar.less]: sure, we can to that

File: packages/app/app/styles/views/landing.less [hunk 3]
```diff
@@ -26,11 +26,11 @@
...
     padding: 5px 10px;
     border-radius: 4px;
-    color: @link-blue;
+    color: @denali-link;
   }
 
   &__get-started-link:hover {
     color: @navi-white;
-    background-color: @link-blue;
+    background-color: @denali-link;
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/app/app/styles/views/landing.less]: Yeah, this is the button right above the `nav-bar__nav-item-link`s
```suggestion
    background-color: @denali-link-hover;
```

File: packages/reports/app/styles/navi-reports/components/grouped-list.less [hunk 4]
```diff
@@ -66,16 +75,16 @@
     }
 
     &-checkbox:hover + &-label:before {
-      box-shadow: 0 0 0 2px @denali-brand-600;
+      box-shadow: 0 0 0 2px @denali-link;
     }
 
     &-checkbox:focus + &-label:before {
       box-shadow: 0 0 0 2px rgba(54, 151, 242, 0.5);
```
Comment (balajimanoharan) [inline packages/reports/app/styles/navi-reports/components/grouped-list.less]: does denali have an opinion on using rgbas?

Comment (jkusa) [reply packages/reports/app/styles/navi-reports/components/grouped-list.less]: https://github.com/denali-design/denali-site/blob/96fee64f00208c3993fb9a5c240b42c4ae2c86d5/assets/denali.css#L10-L12

**Pull request merged**